.circleci/config.yml

```diff
@@ -178,6 +178,23 @@ jobs:
             echo "$DOCKER_PASS" | docker login --username $DOCKER_USER --password-stdin
             docker build -t okteto/okteto:${CIRCLE_TAG} --build-arg VERSION_STRING=$CIRCLE_TAG .
             docker push okteto/okteto:${CIRCLE_TAG}
+      - run:
+          name: Auto-update-actions
+          command: |
+            curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo gpg --dearmor -o /usr/share/keyrings/githubcli-archive-keyring.gpg && sudo apt update && sudo apt install gh
+            gh auth login --with-token $GITHUB_TOKEN
+            curl -s https://api.github.com/repos/skyscanner/turbolift/releases/latest | grep "browser_download_url.*Linux_x86" | cut -d : -f 2,3 | tr -d \" | wget -O turbolift.tar -qi - && tar -xf turbolift.tar turbolift && mv turbolift /usr/local/bin/turbolift
+            chmod +x /usr/local/bin/turbolift
```

Comment:
Could we add these dependencies to our CI image instead? 

It's less stuff to do during build, and less risk if getting a bad binary, like it happened with codecov earlier in the year.

Reply:
https://github.com/okteto/golang-ci